src/styles/main.scss

```
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+$calendar-day-width: 100px;
+$calendar-day-gap: 1px;
+$the-last-day: 31;
+$feb-day-count: 28;
+$week:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-day-gap;
+  width: 7 * $calendar-day-width + 16px;
+
```

here you need to set vertical padding: checklist 11.[STYLES] - Don't be greedy, add some top paddings, so the top row don't go over top of the page on hover.

and from where you get 16px? and you forgot about gaps.Your with should be 
```suggestion
 max-width:($calendar-day-width * $days-in-raw) + ($calendar-day-gap * ( $days-in-raw - 1)) 
```You can realize this with function

---

but it works like that too

---

> but it works like that too

I agree, you don't need to add exactly paddings to center your calendar. But the calculation of width could be more clear:

- Use properly named variables to make all the calculations more clear.

It is best to use function but not necessary. Just make your width calculation include more variables. This solution is good: 
`
 max-width:($calendar-day-width * $days-in-raw) + ($calendar-day-gap * ( $days-in-raw - 1)) ;
`